docs/prompts.md

```diff
@@ -17,18 +17,18 @@ approaches are also supported by working with prompts directly.
 ## Defining prompts
```

Comment:
Let's do something with the paragraph above 

> (Most developers will...)

I propose we scratch it entirely, and then in the prior paragraph, mention that Genkit provides Dotprompt as a way to maintain prompts in source files, etc.

Reply:
Ack. Updated to remove this paragram